src/lib/fetch-data.ts

```diff
@@ -209,3 +212,63 @@ export const fetchTokensPrices = async () => {
     op: { today: Number(opPriceToday), yesterday: Number(opPriceYesterday) },
   };
 };
+
+
+export async function getLatestTransferEvents(
+  contractAddress: Address,
+  page: number = 1,
+  limit: number = 100
+) {
+  console.log(contractAddress);
+  const skip = (page - 1) * limit;
+
+  try {
+    const erc20Transfers = await prisma.erc20Transfer.findMany({
+      where: {
+        address: contractAddress,
+      },
+      orderBy: { transactionHash: 'desc' },
+      take: limit,
+      skip: skip,
+      include: {
+        receipt: {
+          include: {
+            transaction: {
+              include: {
+                block: true,
+              },
+            },
+          },
+        },
+      },
+    });
+
+    const allTransfers = await Promise.all(erc20Transfers.map(async t => {
+      const transactionDate = new Date(Number(t.receipt.transaction.block.timestamp) * 1000);
+
+      return {
+        transactionHash: t.transactionHash,
+        method: t.receipt.transaction.input.slice(0, 10),
+        block: Number(t.receipt.transaction.blockNumber),
+        age: formatDistanceToNow(transactionDate, { addSuffix: true }),
```

Comment:
You can keep the raw timestamp here and use `formatTimestamp` in the table instead.